Resources/Prototypes/tags.yml

```diff
@@ -361,3 +361,5 @@
 - type: Tag
   id: Write
 
+- type: Tag
+  id: NettleProtection #for gloves and gloves only
```

Comment:
This should be generically named.

Reply:
what do you mean by "generically named" ?

Reply:
Something more generic like PointyObjectProtection

Reply:
ah thx